docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -208,6 +208,10 @@ value
   *(optional, string)* Specifies the value of the header. If the value is absent a request that has
   the *name* header will match, regardless of the header's value.
 
+regex
+  *(optional, boolean)* Specifies whether the header value is a regular
```

Comment:
Can you link out to the grammar: http://en.cppreference.com/w/cpp/regex/ecmascript